source/load-balancers/load-balancers.rst

```diff
@@ -0,0 +1,564 @@
+MUST NOT check the connection back into the connection pool until the initial cursor
+or :code:`getMore` command has returned with an :code:`id` of :code:`0`. Otherwise the
+driver MUST continue to use the same connection for all :code:`getMore` and
+:code:`killCursors` commands for a given cursor. Drivers MUST check the connection back into
+pool on all :code:`cursor.close()` calls.
+
+For drivers that implement a connection pool and when using a pinned connection, the driver MUST
+emit only 1 :code:`ConnectionCheckOutStartedEvent`, and only 1 :code:`ConnectionCheckedOutEvent`
+or :code:`ConnectionCheckOutFailedEvent`. Similarly, the driver MUST only publish 1
+:code:`ConnectionCheckedInEvent`.
+
+For single threaded drivers that do not use a connection pool, the driver MUST have only 1
+socket connection to the load balancer in load balancing mode.
+
+In the case of network errors for all operations that create a cursor, the driver MUST behave
+as defined in the Retryable Reads specification. If a getMore fails with a network error,
+drivers MUST unpin the connection that’s pinned to the cursor and return it to the pool
+so it does not count against maxPoolSize. Future calls to close() on the cursor MUST NOT
+attempt to execute a killCursors command on the server.
+
+When the driver is in load balancing mode, drivers MUST NOT check the connection back into
+the connection pool while executing a transaction. The driver MUST continue to use the
+same connection to execute all commands for a given transaction with the exception
+of ::code:`commitTransaction` and :code:`abortTransaction`, if either one of these
+operations fails, the driver MUST unpin connections as follows:
+
+*Drivers MUST unpin a :code:`ClientSession` when a command within a transaction,
+including :code:`commitTransaction` and :code:`abortTransaction`, fails with a
+:code:`TransientTransactionError`. Transient errors indicate that the transaction
+in question has already been aborted or that the pinned mongos is down/unavailable.
+Unpinning the session ensures that a subsequent :code:`abortTransaction`
+(or :code:`commitTransaction`) does not block waiting on a server that is unreachable.
+
+Additionally, drivers MUST unpin a :code:`ClientSession` when any individual
+:code:`commitTransaction` command attempt fails with an :code:`UnknownTransactionCommitResult`
+error label. In cases where the :code:`UnknownTransactionCommitResult` causes an automatic
+retry attempt, drivers MUST unpin the :code:`ClientSession` before performing server selection
+for the retry.
+
+Starting a new transaction on a pinned :code:`ClientSession` MUST unpin the session.
+Additionally, any non-transaction operation using a pinned :code:`ClientSession` MUST unpin
+the session and the operation MUST perform normal server selection.*
+
+In the case of a network errors in transactions, the driver MUST implement the following cases:
```

Comment:
I think we can safely remove any information about network errors in transactions because it should already be covered by the prose in the transactions spec about adding the `TransientTransactionError` label and unpinning the server/connection when an error with this label occurs.

Reply:
Done.

Reply:
I've removed it now.